setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 
 requirements = [
-    'pulpcore-plugin==0.1.0b16',
+    'pulpcore-plugin>=0.1.0b18',
```

Comment:
Are we sure we don't want to pin?

Reply:
After discussion on IRC, we decided to pin releases to pypi, but leave `master` as >=